suite-native/app/package.json

```diff
@@ -1,42 +1,26 @@
 {
-    "name": "@trezor/suite-native",
+    "name": "@suite-native/app",
     "version": "1.0.0",
     "suiteNativeVersion": "24.2.0",
-    "private": true,
+    "main": "index.js",
     "scripts": {
-        "android": "react-native run-android --no-packager --mode=devDebug --appIdSuffix=debug --active-arch-only",
+        "android": "expo run:android",
         "android:clean": "cd android && ./gradlew clean",
-        "ios": "react-native run-ios --no-packager --scheme \"Debug\"",
-        "start": "react-native start --reset-cache",
-        "test": "jest",
```

Comment:
Why do you remove test script?

Reply:
There are no tests in package and script name it's not even in correct form `test:unit` so it's never run anyway.